src/hooks/useFormValidation.ts

```
@@ -2,274 +1,72 @@
+        setErrors(prev => ({ ...prev, [field]: errorMessage }))
+        return errorMessage
+      }
+      return 'Erro de validação'
+    }
+  }
 
-  // Função para submeter o formulário
-  const submit = useCallback(async () => {
-    console.log('🔍 useFormValidation - submit chamado')
-    
+  const validateForm = (data: any, schema: z.ZodSchema): boolean => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)